clusterloader2/README.md

```diff
@@ -1,6 +1,19 @@
 # ClusterLoader
 
-#### Vendor
+### Running ClusterLoader
+
+To run ClusterLoader type:
+```
+go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.json
+```
+Flags clientconfig and testconfig are necessary.
```

Comment:
-> kubeconfig

Reply:
It seems like this hasn't been changed at few places (including main).

Reply:
My bad. This time should be done.